### ISSUE TITLE
String instead of Attribute passed down in AttributeBag

### ISSUE THREAD

**Issue by ghost** [jarektkaczyk/eloquence#32]
I am trying to store an array of data as a meta attribute. This attribute gets stored as JSON with a `meta_type` of `StdClass`.

Now when I retrieve a model directly and retrieve the meta attribute like this `Post::first()->options` everything is working fine. Now comes the weird issue.

When I want to retrieve the model through a relationship and try to access the meta attribute like this `Category::first()->posts()->first()->options` I get the following exception.

```php
Argument 1 passed to Sofa\Eloquence\Metable\AttributeBag::addInstance() must be an instance of Sofa\Eloquence\Metable\Attribute, string given, called in /home/vagrant/Code/homestead/vendor/sofa/eloquence/src/Metable/AttributeBag.php on line 66 and defined
```

When I dump the value that is passed down in Line 66 it is the value of the `meta_value` column instead of an `Attribute` instance.

What could cause this issue? I am trying to figure it out but can't really find anything.

**Comment by jarektkaczyk** [comment]
I will take a look at this.

**Comment by jarektkaczyk** [comment]
@DraperStudio please paste the data and your code. It seems odd that you get different results in this case.

**Comment by DraperStudio** [comment]
```php
Psy Shell v0.5.2 (PHP 5.6.14-1+deb.sury.org~trusty+1 — cli) by Justin Hileman
>>> namespace App;
=> null
>>> Post::first()->options
=> [
     "some_option" => false,
     "another_option" => false,
   ]
>>> Category::first()->posts()->first()->options
=> Argument 1 passed to Sofa\Eloquence\Metable\AttributeBag::addInstance() must be an instance of Sofa\Eloquence\Metable\Attribute, string given, called in /home/vagrant/Code/homestead/vendor/sofa/eloquence/src/Metable/AttributeBag.php on line 66 and defined
>>>
```

**Comment by jarektkaczyk** [comment]
fixed in 5.1.5
